tern/report/report.py

```diff
@@ -154,5 +156,9 @@ def report_out(args, *images):
         logger.error("%s not a recognized plugin.", args.report_format)
     elif args.output_file:
         write_report(report, args)
+        if args.report_format == "json":
```

Comment:
We should add a notice to the user (via the logger) that says something like "Generating <filename>.html which can be viewed in the web browser of your choice".

Reply:
Per our discussion today, let's make the html file generation it's own command line options, so you can ignore this suggestion.